content/release-notes/1.16.0.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2020-06-01T08:00:00-07:00
+linktitle: "1.16.0"
+title: "1.16.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Support for HTTP_PROXY and NO_PROXY environment variables.
+* CronJobs now support private images.
+* Gitops cluster tracers
+* Scheduled update checks
+* Support for custom redactors
```

Comment:
```suggestion
* Support for [custom redactors](https://troubleshoot.sh/reference/redactors/overview/) 
```

Reply:
This is not a 1.16 feature, @laverya this was released in 1.15 IIRC?

Reply:
@divolgin let's remove this bullet